src/go/MONIT/alert.go

```diff
@@ -0,0 +1,658 @@
+			}
+
+		}
+	}
+}
+
+//Function for printing alert's details in JSON format
+func jsonPrintDetails() {
+
+	b, err := json.Marshal(alertDetails[name])
+
+	if err != nil {
+		log.Printf("Unable to convert Detailed JSON Data, error: %v\n", err)
+		return
+	}
+
+	fmt.Println(string(b))
+}
+
+//Function for printing alert's details in Plain text format
+func detailPrint() {
+
+	if each, ok := alertDetails[name].(amGGUSJSON); ok {
+		fmt.Printf("NAME: %s\n", name)
+		fmt.Printf("LABELS\n")
+		fmt.Printf("\tservice: %s\n", each.Labels.Service)
+		fmt.Printf("\ttag: %s\n", each.Labels.Tag)
+		fmt.Printf("\tseverity: %s\n", each.Labels.Severity)
+		fmt.Printf("ANNOTATIONS\n")
+		fmt.Printf("\tpriority: %s\n", each.Annotations.Priority)
+		fmt.Printf("\tresponsible unit: %s\n", each.Annotations.ResponsibleUnit)
+		fmt.Printf("\tscope: %s\n", each.Annotations.Scope)
+		fmt.Printf("\tsite: %s\n", each.Annotations.Site)
+		fmt.Printf("\tstatus: %s\n", each.Annotations.Status)
+		fmt.Printf("\tsubject: %s\n", each.Annotations.Subject)
+		fmt.Printf("\tticketID: %s\n", each.Annotations.TicketID)
+		fmt.Printf("\ttype: %s\n", each.Annotations.Type)
+		fmt.Printf("\tvo: %s\n", each.Annotations.VO)
+		fmt.Printf("\turl: %s\n", each.Annotations.URL)
+	} else if each, ok := alertDetails[name].(amSSBJSON); ok {
+		fmt.Printf("NAME: %s\n", name)
+		fmt.Printf("LABELS\n")
+		fmt.Printf("\tservice: %s\n", each.Labels.Service)
+		fmt.Printf("\ttag: %s\n", each.Labels.Tag)
+		fmt.Printf("\tseverity: %s\n", each.Labels.Severity)
+		fmt.Printf("ANNOTATIONS\n")
+		fmt.Printf("\tdate: %s\n", each.Annotations.Date)
+		fmt.Printf("\tdescription: %s\n", each.Annotations.Description)
+		fmt.Printf("\tfe name: %s\n", each.Annotations.FeName)
+		fmt.Printf("\tmonit state: %s\n", each.Annotations.MonitState)
+		fmt.Printf("\tmonit state1: %s\n", each.Annotations.MonitState1)
+		fmt.Printf("\tse name: %s\n", each.Annotations.SeName)
+		fmt.Printf("\tshort description: %s\n", each.Annotations.ShortDescription)
+		fmt.Printf("\tssb number: %s\n", each.Annotations.SsbNumber)
+		fmt.Printf("\tsys created by: %s\n", each.Annotations.SysCreatedBy)
+		fmt.Printf("\tsys mod count: %s\n", each.Annotations.SysModCount)
+		fmt.Printf("\tsys updated by: %s\n", each.Annotations.SysUpdatedBy)
+		fmt.Printf("\ttype: %s\n", each.Annotations.Type)
+		fmt.Printf("\tupdate timestamp: %s\n", each.Annotations.UpdateTimestamp)
+	} else if each, ok := alertDetails[name].(amGrafanaJSON); ok {
+		fmt.Printf("NAME: %s\n", name)
+		fmt.Printf("LABELS\n")
+		fmt.Printf("\tservice: %s\n", each.Labels.Service)
+		fmt.Printf("\ttag: %s\n", each.Labels.Tag)
+		fmt.Printf("\tseverity: %s\n", each.Labels.Severity)
+		fmt.Printf("ANNOTATIONS\n")
+		fmt.Printf("\tsummary: %s\n", each.Annotations.Summary)
+		fmt.Printf("\tdescription: %s\n", each.Annotations.Description)
+	}
+}
+
+//Function running all logics
+func run() {
+	var amGGUSData ggusData
```

Comment:
read configuration here which will provide which alerts to process, then you can generalize the code.

Reply:
Config read code added.